host-bmc/dbus/serialize.cpp

```diff
@@ -100,5 +100,31 @@ void Serialize::setObjectPathMaps(const ObjectPathMaps& maps)
     }
 }
 
+void Serialize::reSerialize(const std::vector<uint16_t> types)
+{
+    if (types.empty())
+    {
+        return;
+    }
+
+    for (const auto& type : types)
+    {
+        if (savedObjs.contains(type))
+        {
+            savedObjs.erase(savedObjs.find(type));
```

Comment:
before erase, please add this trace :
`std::cerr << "Removing objects of type : " << (unsigned)type << " from the persistent cache\n";`

Reply:
> std::cerr << "Removing objects of type : " << (unsigned)type << " from the persistent cache\n";

Ack